app/views/partials/_navbar.html.haml

```diff
@@ -0,0 +1,9 @@
+.navbar
+  %p= link_to 'Home', root_path
+
+  -if user_signed_in?
+    %p="Hello, #{current_user.first_name}!"
+    %p= link_to 'Log out', '#'
+  -else
+    %p=link_to "Sign up", new_user_registration_path
+    %p= link_to 'Log in', '#'
```

Comment:
`'#'` why this?

Reply:
We have not implemented the log in functionality yet, once that is implemented we will put in the proper path. Currently this is just a placeholder.